src/components/tooltip.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+//possible position of tooltip
+const TOP = 'top';
+const RIGHT = 'right';
+const BOTTOM = 'bottom';
+const LEFT = 'left';
+
+export function Tooltip(props) {
```

Comment:
I think in this case is better to use Default export. I will give the opportunity to import the component without brackets (Tooltip instead of { Tooltip }), that is much nicer :) 

But is not critical, just a comment...